nix/status-go/mobile/default.nix

```diff
@@ -17,11 +17,10 @@
     inherit meta source goBuildLdFlags;
   };
 
-  ios = callPackage ./build.nix {
+  ios = {targets ? [ "ios" "iossimulator"]}: callPackage ./build.nix {
     platform = "ios";
     platformVersion = "8.0";
-    targets = [ "ios" "iossimulator" ];
     outputFileName = "Statusgo.xcframework";
-    inherit meta source goBuildLdFlags;
+    inherit meta source goBuildLdFlags targets;
```

Comment:
Are we not doing a map here? Like we do from Gradle architectures to Gomobile architectures?

Reply:
No, we don't.

iOS `ARCH` is set inside Xcode project file by a combination defferent options. Resulted ARCH can be extracted.

Although in our case we can do a much simpler "mapping":
- if it's a `device` build - build status-go for `ios` target
- if it's a `simulator` build - build status-go for `iossimulator` target

Reply:
Yeah, lets keep it simple.